TRADEMARK.md

```diff
@@ -0,0 +1,96 @@
+# OpenRiverCam Trademark Guidelines
+
+> **_Acknowledgement:_** These guidelines are loosely based on the guidelines for the trademarks of OpenDroneMap (see [this](https://github.com/OpenDroneMap/documents/blob/master/TRADEMARK.md) link for the OpenDroneMap guidelines).
+
+OpenRiverCam is an open source software ecosystem, founded by a collaborative of [Rainbow Sensing](https://rainbowsensing.com/) and the [Trans-African Hydrometeorological Observatory](https://tahmo.org/) within contributions from [OpenDroneMap.org](https://opendronemap.org/) and [Humanitarian OpenStreetMap Team](https://www.hotosm.org/). Maintenance is organised by [Rainbow Sensing](https://rainbowsensing.com/). Because we make our code available to download and modify, our intellectual property (i.e. recognizing who has established and who is maintaining OpenRiverCam) is essential to us. Through these trademark guidelines, we provide users guidance how to refer to our products whilst using one of them, or building a product or service on top of them. When using OpenRiverCam trademarks (including some product names that are part of our organization) you must comply with these OpenRiverCam Trademark Guidelines.
```

Comment:
Typo in line 5: should read "with contributions from..." rather than "with**in** contributions from..."

Reply:
resolved